apps/mobile/src/components/ProfileView/ProfileViewHeader.tsx

```diff
@@ -93,10 +121,13 @@ export function ProfileViewHeader({ queryRef, selectedRoute, onRouteChange }: Pr
         </View>
       )}
       {!isLoggedInUser && <ProfileViewSharedInfo userRef={user} />}
-      <View className={`flex flex-row mx-4 space-x-12 mt-4`}>
-        <ProfileViewTwitterPill userRef={user} />
-        <ProfileViewFarcasterPill userRef={user} />
-      </View>
+      {numPills > 0 && (
+        <View className={`flex flex-row mx-4 mt-4 w-full ml-2`}>
```

Comment:
space-between doesn't work using tailwind in certain cases apparently and it looks like this case is one of them, so I achieved the gap between pills  byusing margin-left on each pill, and a negative margin on the container... 💀

official recommendation lol
https://tailwindcss.com/docs/space#limitations
![CleanShot 2023-08-04 at 21 49 13](https://github.com/gallery-so/gallery/assets/80802871/89d9043d-384a-4b1d-b160-7f4b08e6bf0d)